### DISCUSSION TITLE
Altitude wont go up.

### DISCUSSION THREAD

**Discussion by FlyerMacintosh** [flybywiresim/aircraft#2354] [Help]
The aircraft gets stuck at 10,000 feet and will not go up from there. I have no clue why or a way to fix this. Its extremely annoying. It goes up and then goes down. 

![aa](https://user-images.githubusercontent.com/75852491/101929587-4a23f080-3b94-11eb-8916-a02e763ba806.JPG)

**Comment by Benjozork** [comment]
Can you show a screenshot of your PFD and ND ?
